src/autocropper.py

```diff
@@ -0,0 +1,64 @@
+import cv2
+
+__author__ = 'Thomas'
+
+def iterate_image(img):
+    for i in range(img.shape[0]):
+        for j in range(img.shape[1]):
+            yield (i, j)
```

Comment:
Alternatively, I guess it could also be added to the class. It just seems weird to have a function and a class defined on the same level in the same file.